src/sentry/integrations/pipeline.py

```diff
@@ -94,6 +95,25 @@ def initialize(self) -> None:
         )
 
     def finish_pipeline(self):
+        org_context = organization_service.get_organization_by_id(
+            id=self.organization.id, user_id=self.request.user.id
+        )
+
+        if (
+            org_context
+            and org_context.member
+            and "org:integrations" not in org_context.member.scopes
+        ):
+            logger.info(
+                "build-integration.permission_error",
+                extra={
+                    "error_message": "User has no 'org:integrations' scope.",
+                    "organization_id": self.organization.id,
+                    "user_id": self.request.user.id,
+                    "provider_key": self.provider.key,
+                },
+            )
+
```

Comment:
Should the process stop and raise an error at this point ?

Reply:
I'd like to roll out this "dry" version of this check to make sure we are not breaking any normal flows. If there will be no log entries in a few days, I can append `return self.error("User has no 'org:integrations' scope.")`